server/chat-plugins/friends.ts

```diff
@@ -0,0 +1,910 @@
+/**
+ * Friends list plugin.
+ * Allows for adding and removing friends, as well as seeing their activity.
+ * Written by Mia.
+ * @author mia-pi-git
+ */
+import * as Sqlite from 'better-sqlite3';
+import {Utils} from '../../lib/utils';
+import {FS} from '../../lib/fs';
+import {QueryProcessManager} from '../../lib/process-manager';
+import {Repl} from '../../lib/repl';
+import {Chat} from '../chat';
+import {Dex} from '../../sim';
```

Comment:
Why would a friends list need this import?

Reply:
It's needed for the QueryProcessManager.

Reply:
(Generally the reviewer resolves these when they accept the explanation)